packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -20,6 +20,9 @@ export class AtomStepsModal {
   @Prop() trigger?: string
   @Prop() stepsTitles: string
   @Prop({ mutable: true }) isOpen = false
+  @Prop() primaryText? = 'Próximo'
+  @Prop() secondaryText? = 'Voltar'
+  @Prop() closeOnFinish? = false
```

Comment:
block: if is a optional attribute it can't be a default value, didn't lint catch this? 🤔